client/src/pages/ViewUser.jsx

```diff
@@ -61,50 +61,52 @@ export const ViewUser = observer(() => {
 
   return (
     <PageContainer>
-      <IconButton
-        icon={<ArrowBackIcon />}
-        bg={"transparent"}
-        onClick={() => {
-          navigate(-1);
-        }}
-      />
-      <Stack spacing={4} w={"full"} maxW={"lg"}>
-        <Heading lineHeight={1.1} fontSize={{ base: "2xl", sm: "3xl" }}>
-          User Profile Details
-        </Heading>
-        <SimpleGrid columns={{ base: 1, md: 2 }} spacing={4}>
-          {Object.entries(state).map((item, index) => (
-            <>
-              <Text fontWeight="bold">{item[0]}:</Text>
-              <Text>{item[1]}</Text>
-            </>
-          ))}
-        </SimpleGrid>
-        <Stack spacing={6} direction={["column", "row"]}>
-          <Button
-            bg={"red.400"}
-            color={"white"}
-            w="full"
-            _hover={{
-              bg: "red.500",
-            }}
-            onClick={deleteUser}
-          >
-            Delete your profile
-          </Button>
-          <Button
-            bg={"blue.400"}
-            color={"white"}
-            w="full"
-            _hover={{
-              bg: "blue.500",
-            }}
-            onClick={redirectToUpdateUserPage}
-          >
-            Update your profile
-          </Button>
+      <>
```

Comment:
Oh this was to fix the console error:
"Warning: Failed prop type: Invalid prop `children` of type `array` supplied to `PageContainer`, expected `object`."

It's necessary because `PageContainer` expects its children to be a single object. I'll add a comment in the file explaining why the <><> is necessary